justfile

```diff
@@ -0,0 +1,74 @@
+set dotenv-load
+set positional-arguments
+
+# Adding a chain
+add-chain:
+	go run ./add-chain
+	go run ./add-chain check-rollup-config
+	go run ./add-chain compress-genesis
+	go run ./add-chain check-genesis
```

Comment:
This command has a required flag/env-var for the chain-id

Reply:
[f21bee9](https://github.com/ethereum-optimism/superchain-registry/pull/366/commits/f21bee943b72f74b054e7597810972ca8685afde)